client/my-sites/stats/annual-site-stats/index.js

```diff
@@ -27,58 +29,113 @@ class AnnualSiteStats extends Component {
 						{ numberFormat( data.total_comments ) }
 					</div>
 				</div>
 				<div className="annual-site-stats__stat">
-					<div className="annual-site-stats__stat-title">
-						{ translate( 'avg comments per post' ) }
-					</div>
+					<div className="annual-site-stats__stat-title">{ strings.avg_comments }</div>
 					<div className="annual-site-stats__stat-figure">
 						{ numberFormat( data.avg_comments ) }
 					</div>
 				</div>
 				<div className="annual-site-stats__stat">
-					<div className="annual-site-stats__stat-title">{ translate( 'total likes' ) }</div>
+					<div className="annual-site-stats__stat-title">{ strings.total_likes }</div>
 					<div className="annual-site-stats__stat-figure">{ numberFormat( data.total_likes ) }</div>
 				</div>
 				<div className="annual-site-stats__stat">
-					<div className="annual-site-stats__stat-title">{ translate( 'avg likes per post' ) }</div>
+					<div className="annual-site-stats__stat-title">{ strings.avg_likes }</div>
 					<div className="annual-site-stats__stat-figure">{ numberFormat( data.avg_likes ) }</div>
 				</div>
 				<div className="annual-site-stats__stat">
-					<div className="annual-site-stats__stat-title">{ translate( 'total words' ) }</div>
+					<div className="annual-site-stats__stat-title">{ strings.total_words }</div>
 					<div className="annual-site-stats__stat-figure">{ numberFormat( data.total_words ) }</div>
 				</div>
 				<div className="annual-site-stats__stat">
-					<div className="annual-site-stats__stat-title">{ translate( 'avg words per post' ) }</div>
+					<div className="annual-site-stats__stat-title">{ strings.avg_words }</div>
 					<div className="annual-site-stats__stat-figure">{ numberFormat( data.avg_words ) }</div>
 				</div>
 			</div>
 		);
 	}
 
+	renderTable( data, strings ) {
+		const keys = Object.keys( strings );
+		return (
+			<div className="annual-site-stats__table-wrapper module-content-table">
+				<div className="annual-site-stats__table-scroll module-content-table-scroll">
+					<table cellPadding="0" cellSpacing="0">
+						<thead>
+							<tr>
+								{ keys.map( key => (
+									<th scope="col" key={ key }>
+										{ strings[ key ] }
+									</th>
+								) ) }
+							</tr>
+						</thead>
+						<tbody>
+							{ data.map( ( row, i ) => (
+								<tr key={ i }>
+									{ keys.map( ( key, j ) => {
+										const Cell = j === 0 ? 'th' : 'td';
+										return (
+											<Cell scope={ j === 0 ? 'row' : null } key={ j }>
+												{ row[ key ] }
+											</Cell>
+										);
+									} ) }
+								</tr>
+							) ) }
+						</tbody>
+					</table>
+				</div>
+			</div>
+		);
+	}
+
+	getStrings() {
+		const { translate } = this.props;
+		return {
+			year: translate( 'Year' ),
+			total_posts: translate( 'Total Posts' ),
+			total_comments: translate( 'Total Comments' ),
+			avg_comments: translate( 'Avg Comments per Post' ),
+			total_likes: translate( 'Total Likes' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'total likes' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).